pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -839,6 +843,7 @@ func propagateError(higherLayerPort, lowerLayerPort *types.NetworkPortConfig) {
 func propagatePhyioAttrsToPort(port *types.NetworkPortConfig, phyio *types.PhysicalIOAdapter) {
 	port.Phylabel = phyio.Phylabel
 	port.IfName = phyio.Phyaddr.Ifname
+	port.Cbattr = phyio.Cbattr
```

Comment:
We also have some compare functions (in types package I think) which compare the phyio fields. They might need to compare Cbattr if they don't already.

Reply:
Sounds like that is the only comparison. I also seem two places in assignableadapters.go where we do assignment/copying of fields like the Serial etc.

Reply:
But I suspect it is easier to have the code in parseconfig look at the map and fill in specific fields similarly to how we handle pcilong etc from Phyaddrs. Having the maps in the API makes it easier to add new stuff, but since we need to implement specific semantics in EVE-OS for this new stuff we might as well define specific fields in the golang struct(s) for them.

Reply:
Agree - in the internal Go structures there is no need to continue working with cbattr as a map, we can add VFCount integer into `NetworkPortConfig`. The reason why `cbattr` is a map in the APIs is because `PhysicalIO` can be any IO, not just NIC, so we need to have it more flexible.